sphinx_automodapi/automodsumm.py

```diff
@@ -16,12 +16,18 @@
     * ``:classes-only:``
         If present, the autosummary table will only contain entries for
         classes. This cannot be used at the same time with
-        ``:functions-only:`` .
+        ``:functions-only:`` or ``:global-variables-only:``.
 
     * ``:functions-only:``
         If present, the autosummary table will only contain entries for
         functions. This cannot be used at the same time with
-        ``:classes-only:`` .
+        ``:classes-only:`` or ``:global-variables-only:``.
+
+    * ``:global-variables-only:``
```

Comment:
@mhvk - just out of curiosity, why 'global-variables-only' rather than simply 'variables-only'? Using 'global' is confusing since it's not actually going to include everything in ``globals()``